addons/mail/static/src/components/attachment_viewer/attachment_viewer.xml

```diff
@@ -21,73 +21,73 @@
                             </t>
                         </div>
                     </t>
-                    <div class="o_AttachmentViewer_headerItem o_AttachmentViewer_name">
+                    <div class="o_AttachmentViewer_headerItem d-flex align-items-center mx-2 o_AttachmentViewer_name">
```

Comment:
Again, move utility-classes **after** odoo's custom classes... could you check all the other lines?